lib/constructs/interface.js

```diff
@@ -492,8 +492,7 @@ class Interface {
   }
 
   generateRequires() {
-    this.requires.addRaw("impl", "utils.implSymbol");
-    this.requires.addRaw("ctorRegistry", "utils.ctorRegistrySymbol");
+    this.requires.addRaw(`{ implSymbol, ctorRegistrySymbol }`, "utils");
```

Comment:
I’ve changed this to use object destructuring to match how alias imports are done in **JSDOM**.